boost-maven/boost-boms/boost-mp20-apis-bom/pom.xml

```diff
@@ -38,26 +31,11 @@
 
 	<dependencyManagement>
```

Comment:
For MP API boms, we should just import existing Eclipse BOMs from project:  https://github.com/eclipse/microprofile/blob/master/pom.xml.   A couple seem to also define EE API versions, which means we should start by NOT importing EE API BOM, until we realize we have a reason to do so.